AlphaWallet/Transactions/ViewControllers/TransactionsViewController.swift

```diff
@@ -3,88 +3,80 @@
 import UIKit 
 import StatefulViewController
 import AlphaWalletFoundation
+import Combine
 
 protocol TransactionsViewControllerDelegate: AnyObject {
     func didPressTransaction(transactionRow: TransactionRow, in viewController: TransactionsViewController)
 }
 
 class TransactionsViewController: UIViewController {
-    private var viewModel: TransactionsViewModel
-    private let tableView = UITableView(frame: .zero, style: .grouped)
+    private let viewModel: TransactionsViewModel
```

Comment:
Some commented out code above. Would you describe the change? "Update transactions screen" is quite vague :)

Reply:
yes, pr update TransactionViewController and its view model to observe changes:
- when changes token rate  
- when changes chain state (e.g block number)
- when changed transaction (we opdate transaction object in db, updating its status, and other info)

current implementation is static, transaction screen don't get updated when some of transaction data has changed.